library/std/src/io/error.rs

```diff
@@ -199,6 +204,7 @@ impl ErrorKind {
             ErrorKind::TimedOut => "timed out",
             ErrorKind::WriteZero => "write zero",
             ErrorKind::Interrupted => "operation interrupted",
+            ErrorKind::NoStorageSpace => "no storage space",
```

Comment:
```
$ errno ENOSPC
ENOSPC 28 No space left on device
```
```suggestion
            ErrorKind::NoStorageSpace => "no space left on device",
```

Reply:
Thanks for the suggestion.  I'm happy to change the message to be identical to the traditional Unix one if that seems better.  But most of the existing messages do not mirror the traditional Unix ones.